tree/30_generic_methods/service_check_disabled_at_boot.cf

```diff
@@ -0,0 +1,80 @@
+#####################################################################################
+# Copyright 2017 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Service check disabled at boot
+# @description Check if a service is set to not start at boot using the appropriate method
+#
+# @parameter service_name  Service name (as recognized by systemd, init.d, etc...)
+# 
+# @class_prefix service_check_disabled_at_boot
+# @class_parameter service_name
+# This bundle will define a class service_check_disabled_at_boot_${service_name}_{kept,ok,not_ok,failed,reached}
+
+bundle agent service_check_disabled_at_boot(service_name)
+{
+  vars:
+
+    pass1.systemctl_utility_present::
+      "command_to_check"   string => "${paths.path[systemctl]} is-enabled ${service_name} | grep disabled";
```

Comment:
shouldn't the grep be quiet ?
Or even better, exit code of systemctl is 1 for disabled, 0 for enabled